content/docs/dispensary/prescriptions.md

```diff
@@ -238,117 +245,77 @@ In the `Add Item` window, you can look up an item by:
+| :-------------------- | :-------------------------------------------------------------------------------------- |
+| **Expiry**            | Expiry date of the batch (format: MM/YYYY)                                              |
+| **Batch**             | Batch number. It is a designation given to products made in the same manufacturing run. |
+| **Pack Size**         | Quantity of units per pack                                                              |
+| **In Stock (units)**  | Total number of units in your store                                                     |
+| **Available (units)** | Number of units available (not already allocated to other invoices)                     |
+| **Units Issued**      | Total quantity of units to be issued                                                    |
 
 ![List of available batch numbers](/docs/dispensary/images/prescription_additem_listofbatches.png)
 
-The batches which are available for selection are shown first and batches which are not (on hold or no stock available) are shown below and in a grey font.
-
-### Issue a quantity of units
-
-By default, you are invited to issue a quantity of **units**, regardless of the pack size (number of units in a pack).
+### Issue a quantity
 
 <div class="imagetitle">
-In the below example, we are issuing 1000 units of the item. All 1000 units will be taken from the same batch (third row). The first two batches was not selected because they are expired. 
+In the below example, we are issuing 1000 units of the item. All 1000 units will be taken from the same batch (first row). The other two batches was not selected because they are expired or on hold.
 </div>
 
-![additem!](/docs/dispensary/images/prescription_additem_issueunits.png)
+![Auto allocate 1000 units from single batch](/docs/dispensary/images/prescription_additem_1.png)
+![Auto allocate 1000 units from single batch (Collapsed View)](/docs/dispensary/images/prescription_additem_1_collapsed.png)
 
 <div class="imagetitle">
-In this other example, we are issuing 82 units. System selects two batches with different pack sizes. 
+In this other example, we are issuing 11 units. System selects two batches with different pack sizes. 
 </div>
 
-![additem!](/docs/dispensary/images/prescription_additem_issueunitstwobatches.png)
+![Allocated Item 2 Batches selected](/docs/dispensary/images/prescription_additem_issueunitstwobatches.png)
+![Allocated Item 2 Batches selected (Collapsed View)](/docs/dispensary/images/prescription_additem_issueunitstwobatches_collapsed.png)
 
 #### Warnings
 
 You may see one or more warning messages on this screen:
 
-![placeholder warning!](/docs/distribution/images/os_warning_placeholder.png)
-
-The warning is shown to let you know that not enough stock is available to fulfil your request. A placeholder quantity is added automatically so that you can keep track of the amount requested, while adding more stock (for example by using an Internal Order or Inbound Shipment).
```

Comment:
We seem to have got rid of the warning that you don't have enough stock. @CarlosNZ do we plan to put it back? Would be good the tell the user why they can't allocate that number?

Reply:
![Screenshot 2025-01-08 at 5 26 45 PM](https://github.com/user-attachments/assets/34f585c3-fac9-4899-a718-9444406efae7)

The warning is still there. (You might have done this on an earlier branch before I'd made sure they were working properly)

Reply:
![Screenshot 2025-01-08 at 5 26 45 PM](https://github.com/user-attachments/assets/34f585c3-fac9-4899-a718-9444406efae7)

The warning is still there. (You might have done this on an earlier branch before I'd made sure they were working properly)

Reply:
thanks! Will update with that screenshot

Reply:
I wonder if we can tweak the wording of the string as it's not really related to packsizes in this case...